src/js/controllers/index.js

```diff
@@ -42,16 +41,28 @@ angular.module('copayApp.controllers').controller('indexController', function($r
     go.walletHome();
   });
 
+  $rootScope.$on('newAddress', function(event, addr) {
+    self.addr = addr;
+  });
+
+  $rootScope.$on('updateTransactionHistory', function(event, h) {
```

Comment:
the other 'update*' event handlers call `fc.get*` functions.

This one only assigns data. It should be called differently.